.github/workflows/build-cirros-bionic.yaml

```diff
@@ -0,0 +1,189 @@
+name: CirrOS image builder
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-cirros-aarch64:
+    name: aarch64
+    runs-on: ubuntu-18.04
+    env:
+      ARCHES: aarch64
+      BOOTTEST: "true"
+      QUIET: 1
+    steps:
+      - name: Pull cirros source artifacts
+        uses: actions/checkout@v3
+
+      - name: Build cirros-aarch64
+        shell: bash
+        run: |
+          sudo apt-get install -y cloud-utils qemu-system openbios-ppc
+          ./bin/system-setup
+          ./bin/build-release daily
+          rm -rf ../build-*/build ../build-*/stage ../build-*/tmp
+          tar -cJf cirros-aarch64.tar.xz ../build-*
```

Comment:
Repeating this for all arches seems a bit tedious, maybe instead add a new wrapper script that just takes the arch as parameter and also handles setting the environment?

Reply:
Looks like we can just use a matrix as mentioned in a later comment by hrw. I've mostly just seen them used for Python but it looks like it can help us here as well.

Reply:
I use that matrix one in my repo. Works fine, cancels whole job if any of builds fail.